peernotifier/log.go

```diff
@@ -0,0 +1,29 @@
+package peernotifier
+
+import (
+	"github.com/btcsuite/btclog"
+	"github.com/lightningnetwork/lnd/build"
+)
+
+// log is a logger that is initialized with no output filters.  This
+// means the package will not perform any logging by default until the caller
+// requests it.
+var log btclog.Logger
+
+// The default amount of logging is none.
+func init() {
+	UseLogger(build.NewSubLogger("PEER", nil))
```

Comment:
I think `PEER` is most likely already in use?

Reply:
It is, but from what I can tell it won't be used at all except in unit tests because the `genSubLogger` param is nil? 

More generally, I wasn't sure whether to make a new sublogger in `lnd/log.go` for peer notifier (it felt a bit much since it's only used a handful of times) so I reused the peer logger. The comments in `lnd/log.go` say new logger "per subsystem"?

Reply:
i'm okay with using the new `PRNF` tag for now, we can consolidate later if need be